### PR TITLE
common: support changing vertical speed in DO_CHANGE_SPEED

### PR DESCRIPTION
**Pull request by chobitsfan**
https://github.com/ArduPilot/ardupilot/pull/9535

It allows users to change vertical speed in auto mission using DO_CHANGE_SPEED. Users can set param1 to 2 for changing climb speed and set param1 to 3 for changing descent speed.

### CODE REVIEW

**Comment by amilcarlucas** [comment]
It's implemented in Ardupilot.
Is it OK to merge?

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -1065,7 +1065,7 @@
       </entry>
       <entry value="178" name="MAV_CMD_DO_CHANGE_SPEED">
         <description>Change speed and/or throttle set points.</description>
-        <param index="1">Speed type (0=Airspeed, 1=Ground Speed)</param>
+        <param index="1">Speed type (0=Airspeed, 1=Ground Speed, 2=Climb speed, 3=Descent speed)</param>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: - capitalisation of "speed" - ie either Ground speed OR Climb Speed, Descent Speed.

Comment (chobitsfan) [reply message_definitions/v1.0/common.xml]: I got it, Thank you

**Comment by hamishwillee** [comment]
It looks good to me, but not been swept up by project ownership. I've therefore added to dev call and hope I can get it approved tonight.

**Comment by chobitsfan** [comment]
hi @hamishwillee Thank you

**Comment by hamishwillee** [comment]
@LorenzMeier Can you please review this.

**Comment by LorenzMeier** [comment]
Thanks!

**Pull request merged**